test/model-property.js

```diff
@@ -9,10 +9,12 @@ var ModelProperty = app.models.ModelProperty;
 var ModelDefinition = app.models.ModelDefinition;
 var ConfigFile = app.models.ConfigFile;
 var TestDataBuilder = require('./helpers/test-data-builder');
+var request = require('supertest');
 
 describe('ModelProperty', function() {
   beforeEach(givenBasicWorkspace);
   beforeEach(function(done) {
+    this.modelId = 'server.user';
```

Comment:
any reason we use `this.modelId` here instead of a `var modelId = 'server.user'`? also noticed other similar format in this test file.